strawberry/utils/typing.py

```diff
@@ -117,3 +117,9 @@ def __dataclass_transform__(
     field_descriptors: Tuple[Union[type, Callable[..., Any]], ...] = (()),
 ) -> Callable[[_T], _T]:
     return lambda a: a
+
+
+def update_annotations(cls: Type, annots: dict) -> None:
```

Comment:
Possible to narrow the `dict` type annotation? I believe the key will always be a `str`?